### PR TITLE
DSC19-22 update start script and docs

### PR DESCRIPTION
**Pull request by MattyJ007**

### CODE REVIEW

File: docs/docs/how-to/creating-packages.mdx [hunk 1]
```diff
@@ -115,4 +115,77 @@ The Instant OpenHIE executable will look for these scripts and ensure that they
...
+
+To add multiple custom packages, list each package location with the flag.
+
+`-c="/path/to/package_1" -c="/path/to/package_2"`
+
+Below is an example of an entire `init` command.
+If you had downloaded the [who-covid19-surveillance-package](https://github.com/jembi/who-covid19-surveillance-package) repository onto your machine you could reference it as follows:
+
+```sh
+yarn docker:instant init core covid19surveillance -c="../who-covid19-surveillance-package"
```
Comment (rcrichton) [inline docs/docs/how-to/creating-packages.mdx:149]: Maybe we should mention that they could also just copy the `deploy.sh` script to their own repository if they didn't want to clone the entire instant repo. That might be a better way for 3rd party package developers not to have to worry about the instant repo at all.

**Review by rcrichton — COMMENTED** [review]
Looks awesome @MattyJ007!! Just one comment to consider.

**Review by rcrichton — COMMENTED** [review]
Looks awesome @MattyJ007!! Just one comment to consider.

File: docs/docs/how-to/creating-packages.mdx (hunk 1 above)
Comment (MattyJ007) [reply docs/docs/how-to/creating-packages.mdx:149]: Good plan will add that now 👍🏼

File: docs/docs/how-to/creating-packages.mdx [hunk 2]
```diff
@@ -115,4 +115,110 @@ The Instant OpenHIE executable will look for these scripts and ensure that they
...
+
+The Instant OpenHIE project is available as a Docker image therefore we do not need the whole GitHub repository to run the containers.
+
+For a minimum Instant OpenHIE set up, download [this deploy script from GitHub](https://raw.githubusercontent.com/openhie/instant/master/deploy.sh).
+Once downloaded make sure it's executable: `sudo chmod +x deploy.sh`
+
+Then, run the following command to add your custom package and initialise the system in docker.
+
+```sh
+./deploy init -t docker core covid19surveillance -c="../who-covid19-surveillance-package"
```
Comment (rcrichton) [inline docs/docs/how-to/creating-packages.mdx:142]: Maybe we should make this more generic?

```suggestion
./deploy init -t docker <your_package_ids> -c="../path/to/your/package"
```

Comment (rcrichton) [reply docs/docs/how-to/creating-packages.mdx:142]: This might also be useful elsewhere in these docs.

**Review by rcrichton — APPROVED** [review]
@MattyJ007 looks good to me, I have a comment but feel free to merge once addressed (if you agree).

**Pull request merged**